modules/consensus/synchronize.go

```diff
@@ -237,6 +242,11 @@ func (cs *ConsensusSet) threadedReceiveBlocks(conn modules.PeerConn) (returnErr
 // that BlockHeight onwards are returned. It also sends a boolean indicating
 // whether more blocks are available.
 func (cs *ConsensusSet) rpcSendBlocks(conn modules.PeerConn) error {
+	if err := cs.tg.Add(); err != nil {
+		return err
+	}
+	defer cs.tg.Done()
```

Comment:
I believe the host has them as `threadedRPC`, if you make it so that the prefix `rpc` implies `threaded` you've added to the vocabulary and introduced a risk of functions in the gateway that prefix with `rpc` but aren't actually RPCs, something like `rpcFactory` or whatever (forgive the crappy example).

Better to have a smaller number of keywords.